src/ResourceManagement/EventHub/Generated/Models/TrackedResourceInner.cs

```diff
@@ -19,24 +18,31 @@ namespace Microsoft.Azure.Management.EventHub.Fluent.Models
     /// <summary>
     /// Definition of Resource
     /// </summary>
-    public partial class TrackedResourceInner : Resource
+    public partial class TrackedResourceInner : Management.ResourceManager.Fluent.Resource
     {
         /// <summary>
         /// Initializes a new instance of the TrackedResourceInner class.
         /// </summary>
         public TrackedResourceInner()
-            : base()
         {
+            CustomInit();
         }
 
         /// <summary>
         /// Initializes a new instance of the TrackedResourceInner class.
         /// </summary>
         /// <param name="location">Resource location</param>
         /// <param name="tags">Resource tags</param>
-        public TrackedResourceInner(string location = default(string), string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>))
+        public TrackedResourceInner(string id = default(string), string name = default(string), string type = default(string), string location = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>))
```

Comment:
since all these are string type params… can you make sure that the caller is now calling with the correct sequence as well?

Reply:
Just checked, these are auto generated code and the callers have the right parameter sequences.